internal/pkg/aws/ecs/errors.go

```diff
@@ -3,7 +3,42 @@
 
 package ecs
 
-import "errors"
+import (
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+const (
+	fmtErrContainerStopped = "task %s: %s"
+)
 
 // ErrNoDefaultCluster occurs when the default cluster is not found.
 var ErrNoDefaultCluster = errors.New("default cluster does not exist")
+
+// ErrResourceNotReadyForTasks contains the STOPPED reason for the container of the first task that failed to start.
+type ErrWaiterResourceNotReadyForTasks struct {
+	tasks []*Task
+	awsErrResourceNotReady error
+}
+
+func (e *ErrWaiterResourceNotReadyForTasks) Error() string {
+	for _, task := range e.tasks {
+		if aws.StringValue(task.LastStatus) != DesiredStatusStopped {
+			continue
+		}
+
+		// TODO: generalize this to be an essential container.
```

Comment:
nit: any

Reply:
ouchh it's merged. i will fix it in the other open PR